crates/sui-core/src/authority.rs

```diff
@@ -2572,6 +2591,20 @@ impl AuthorityState {
         Ok(new_epoch_store)
     }
 
+    pub fn checkpoint_perpetual_db(&self) -> SuiResult<PathBuf> {
+        let _metrics_guard = self.metrics.state_snapshot_checkpoint_latency.start_timer();
+        let checkpoint_path = PathBuf::from(format!(
+            "epoch_{}",
```

Comment:
can we name this `perpetual_store_snapshot_epoch_{}` so that it is not confused with the epoch store?